examples/window.rs

```diff
@@ -17,17 +18,29 @@ fn main() {
         .build(&event_loop)
         .unwrap();
 
+    let mut graphics_context = unsafe { GraphicsContext::new(window) }.unwrap();
+
     event_loop.run(move |event, _, control_flow| {
         control_flow.set_wait();
         println!("{:?}", event);
 
         match event {
+            Event::RedrawRequested(window_id) if window_id == graphics_context.window().id() => {
+                let (width, height) = {
+                    let size = graphics_context.window().inner_size();
+                    (size.width, size.height)
+                };
+                let colour = 0xffee36;
+                let mut buffer = Vec::new();
+                buffer.resize(width as usize * height as usize, colour);
```

Comment:
should use `Vec::with_capacity()` to create a properly sized buffer upfront.

Reply:
If we're talking about optimizing. Doing `vec![color; (width * height) as usize]` is the way to go.

Reply:
@filnet it only allocates once so no difference. @kchibisov is of course neater.

Reply:
> @filnet it only allocates once so no difference.

I learned something today.